.gitignore

```diff
@@ -3,3 +3,4 @@ vendor
 phpunit.xml
 coverage
 _build
+.idea
```

Comment:
Can you please remove this added line as it is not project specific and should be added to your global `.gitignore` file